app/i18n-node.ts

```diff
@@ -0,0 +1,21 @@
+import i18n from 'i18n';
```

Comment:
This is the standard `i18n` for node -- with this, everything in the menu works OK. I still don't know why i18next doesn't produce translations, so I went for using `i18n` instead for the menu.